php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -1469,17 +1469,14 @@ abstract class NDB_BVL_Instrument extends NDB_Page
     function getSessionID(): int
     {
         $query      = "SELECT SessionID FROM flag WHERE CommentID = :CID";
-        $fieldValue = \Database::singleton()->pselectOne(
+        $fieldValue = \Database::singleton()->pselectOneInt(
             $query,
             [
                 'CID' => $this->getCommentID(),
             ]
         );
 
-        if (!empty($fieldValue)) {
-            return $fieldValue;
-        }
-        return -1;
+        return $fieldValue == 0 ? $fieldValue : -1;
```

Comment:
@johnsaigle in your way, it only allows return 0 or -1.
  It shuld be like this   [  return $fieldValue !== null ? $fieldValue : -1;   ]

Reply:
Thanks! I'll try that

Reply:
Hmm it looks like this function was always supposed to return an int in the function signature. If it returned null before there would've been an error. So I'm not sure this is the problem.

Reply:
@kongtiaowang @johnsaigle I slightly updated the return statement and PR is passing Travis now.